README.md

```diff
@@ -20,17 +20,13 @@ For a simple Docker-based install of the Counterparty software stack, see [this
 
 # Manual installation
 
-Download the newest [patched Bitcoin Core](https://github.com/btcdrak/bitcoin/releases) and create
-a `bitcoin.conf` file with the following options:
+Download and install [BCoin](http://bcoin.io/) and create
+a `bcoin.conf` file with the following options:
```

Comment:
Please remove all of the bcoin updates for this pull request.  We should move bcoin support to a separate PR.